pkg/sriov/networknodestate_test.go

```diff
@@ -328,6 +328,42 @@ func TestNetworkNodeStateGetNumVFs(t *testing.T) {
 	}
 }
 
+func TestNetworkNodeStateGetTotalVFs(t *testing.T) {
+	testCases := []struct {
+		netInterface srIovV1.InterfaceExts
+		upInterface  srIovV1.InterfaceExts
```

Comment:
I don't see it being used anywhere. Needed?

Reply:
removed